hmiddleware/httpmetrics/httpmetrics_v2.go

```diff
@@ -0,0 +1,74 @@
+package httpmetrics
+
+import (
+	"net/http"
+	"strconv"
+	"time"
+
+	"github.com/heroku/x/go-kit/metrics"
+	"github.com/heroku/x/go-kit/metricsregistry"
+
+	"github.com/go-chi/chi"
+	"github.com/go-chi/chi/middleware"
+)
+
+const (
+	// metric names
+	requestDuration = "http.server.duration"        // duration in milliseconds
+	activeRequests  = "http.server.active_requests" // counter for number of requests
+
+	// metric attribute keys
+	routeKey         = "http.route"
+	methodKey        = "http.request.method"
+	statusKey        = "http.response.status_code"
+	serverAddressKey = "server.address"
+	urlSchemeKey     = "url.scheme"
```

Comment:
Nice, this looks it is following guidelines from https://opentelemetry.io/docs/specs/otel/metrics/semantic_conventions/http-metrics/#http-server

Reply:
Yes, @slizco pointed this convention out to me. I tried to utilize the [attribute extraction functions](https://pkg.go.dev/go.opentelemetry.io/otel@v1.0.0-RC3/semconv/v1.4.0#HTTPServerMetricAttributesFromHTTPRequest) but they end up adding attributes that would end up with high cardinality.